test/e2e/scenarios/check-your-appeal/checkYourAppeal.js

```diff
@@ -1,25 +1,34 @@
 'use strict';
 
 const paths = require('paths');
-const content = require('steps/check-your-appeal/content.en.json');
 
 Feature('Check-your-appeal');
 
 Before((I) => {
     I.createTheSession();
+    I.enterBenefitTypeAndContinue('pip');
     I.amOnPage(paths.checkYourAppeal);
 });
 
 After((I) => {
     I.endTheSession();
 });
 
-Scenario('When I click submit your appeal, I am taken to the confirmation page', (I) => {
+//TODO: Please, review this test since the wording does not seem to match the test implementation
```

Comment:
The reason the implementation doesn't match the wording is because you have changed the precondition in the `Before` hook by adding `I.enterBenefitTypeAndContinue('pip');` Is there a good reason for adding this?

Reply:
done